src/components/Providers/ProviderInfo/ProviderInfo.js

```diff
@@ -0,0 +1,44 @@
+import Card from '../../UI/Card/Card';
+import ProviderBio from '../ProviderBio/ProviderBio';
+import ProviderDetails from '../ProviderDetails/ProviderDetails';
+import Button from '../../UI/Button/Button';
+import Divider from '../../UI/Divider/Divider';
+import { formatProviderHeading } from '../../../helpers/utils';
+import './ProviderInfo.scss';
+
+const ProviderInfo = ({ provider }) => {
+  const { name, title, designation, bio, location, education, languages } =
+    provider;
+
+  const providerHeading = formatProviderHeading(name, title);
+
+  const bookAppointment = () => {
+    alert('Book an appointment!');
```

Comment:
The text is in the app hard-coded. What changes would you need to make to support French users?